### ISSUE TITLE
Certain ARIA roles must contain particular children

### ISSUE THREAD

**Issue by khattakdev** [bryntum/support#12114]
1. Go to: https://bryntum.com/products/gantt/examples/advanced/
2. Scan with page with axe DevTools
Results:
16 instances of this failure for https://dequeuniversity.com/rules/axe/4.10/aria-required-children

[Issue-2.pdf](https://github.com/user-attachments/files/23159383/Issue-2.pdf)

**Comment by ahmadzaheer-dev** [comment]
[+1 here](https://forum.bryntum.com/viewtopic.php?f=44&t=35669&p=181386#p181386)
